content/expressions/methods.md

```diff
@@ -115,7 +115,7 @@ class Bar
 
 ```
 
-Constructors can also be called on an expression. Here an object is created of the same type as the specified expression - this is equivalent to directly specifying the type.
+Constructors can also be called on an expression. Here an object is created of the same type as the specified expression -- this is equivalent to directly specifying the type.
```

Comment:
Could you revert the hyphen addition? As it isn't related to the issue.

Reply:
Absolutely. Force of habit perpetually fixing such things in my own work. Will get the change made later today.

Reply:
@EpicEric the hyphen is now removed